docs/TroubleshootingGuide.md

```diff
@@ -0,0 +1,550 @@
+    <calloutId>279</calloutId>
+</item>
+<item>
+    <filterKey>{162acb09-0cd9-4b80-b7a7-bdd653cca03a}</filterKey>
+    <displayData>
+        <name>net eBPF sock_addr hook</name>
+        <description>net eBPF sock_addr hook WFP filter</description>
+    </displayData>
+    <flags/>
+    <providerKey>{ddb851f5-841a-4b77-8a46-bb7063e9f162}</providerKey>
+    <providerData/>
+    <layerKey>FWPM_LAYER_ALE_CONNECT_REDIRECT_V6</layerKey>
+    <subLayerKey>{98849e12-b07d-11ec-9a30-18602489beee}</subLayerKey>
+    <weight>
+        <type>FWP_EMPTY</type>
+    </weight>
+    <filterCondition/>
+    <action>
+        <type>FWP_ACTION_CALLOUT_TERMINATING</type>
+        <filterType>{98849e10-b07d-11ec-9a30-18602489beee}</filterType>
+    </action>
+    <rawContext>18446624845314639248</rawContext>
+    <reserved/>
+    <filterId>68246</filterId>
+    <effectiveWeight>
+    <type>FWP_UINT64</type>
+    <uint64>0</uint64>
+    </effectiveWeight>
+</item>
+</filters>
+```
+
+This is the expected `callout` and `filter` at the `FWPM_LAYER_ALE_AUTH_CONNECT_V4` layer:
+```
+<item>
+    <calloutKey>{98849e0b-b07d-11ec-9a30-18602489beee}</calloutKey>
+    <displayData>
+        <name>ALE Authorize Connect eBPF Callout v4</name>
+        <description>ALE Authorize Connect callout for eBPF</description>
+    </displayData>
+    <flags numItems="1">
+        <item>FWPM_CALLOUT_FLAG_REGISTERED</item>
+    </flags>
+    <providerKey>{ddb851f5-841a-4b77-8a46-bb7063e9f162}</providerKey>
+    <providerData/>
+    <applicableLayer>FWPM_LAYER_ALE_AUTH_CONNECT_V4</applicableLayer>
+    <calloutId>274</calloutId>
+</item>
+<item>
+    <filterKey>{f202cbe9-da2b-41bc-8db0-b25a799531b5}</filterKey>
+    <displayData>
+        <name>net eBPF sock_addr hook</name>
+        <description>net eBPF sock_addr hook WFP filter</description>
+    </displayData>
+    <flags/>
+    <providerKey>{ddb851f5-841a-4b77-8a46-bb7063e9f162}</providerKey>
+    <providerData/>
+    <layerKey>FWPM_LAYER_ALE_AUTH_CONNECT_V4</layerKey>
+    <subLayerKey>{7c7b3fb9-3331-436a-98e1-b901df457fff}</subLayerKey>
+    <weight>
+        <type>FWP_EMPTY</type>
+    </weight>
+    <filterCondition/>
+    <action>
+        <type>FWP_ACTION_CALLOUT_TERMINATING</type>
+        <filterType>{98849e0b-b07d-11ec-9a30-18602489beee}</filterType>
+    </action>
+    <rawContext>18446624845314639248</rawContext>
+    <reserved/>
+    <filterId>68244</filterId>
+    <effectiveWeight>
+    <type>FWP_UINT64</type>
+    <uint64>0</uint64>
+    </effectiveWeight>
+</item>
+```
+
+--------------------
+
+## Verify the eBPF Program is Configured Correctly
+
+1. [Verify the eBPF program is loaded](#verify-the-ebpf-program-is-loaded)
+2. [Verify the eBPF program is attached](#verify-the-ebpf-program-is-attached)
+3. [Verify eBPF maps are properly configured](#verify-ebpf-maps-are-properly-configured)
+4. [eBPF Program Load or Attach Failures](#ebpf-program-load-or-attach-failures)
```

Comment:
All the other bullets are written as an instruction to the user.  "eBPF Program Load or Attach Failures" is not an instruction to the user.